web/src/app/[locale]/(app)/donations/event/page.tsx

```diff
@@ -0,0 +1,187 @@
+      requiredItems: [
+        { item: 'Food Kits', quantity: 150 },
+        { item: 'Instant Noodles', quantity: 400 },
+      ],
+    },
+    {
+      title: 'Meals for Orphanages',
+      description: 'Provide nutritious meals to orphanages in need.',
+      image:
+        'https://plus.unsplash.com/premium_photo-1683140523610-13deecbd20b1?q=80&w=2574&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+      altText: 'Orphanage Meals',
+      requiredItems: [
+        { item: 'Meal Boxes', quantity: 300 },
+        { item: 'Juice Boxes', quantity: 200 },
+      ],
+    },
+  ],
+  money: [
+    {
+      title: 'Financial Aid for Earthquake Survivors',
+      description: 'Support those rebuilding their lives after the recent earthquake.',
+      image:
+        'https://plus.unsplash.com/premium_photo-1716717998303-803fdbc38747?q=80&w=2532&auto=format&fit=crop',
+      altText: 'Earthquake Relief',
+      targetAmount: 10000,
+      donatedAmount: 7000,
+    },
+    {
+      title: 'Support Education for Children in Need',
+      description: 'Donate to provide scholarships and resources for underprivileged kids.',
+      image:
+        'https://images.unsplash.com/photo-1509062522246-3755977927d7?q=80&w=2464&auto=format&fit=crop',
+      altText: 'Education Support',
+      targetAmount: 6000,
+      donatedAmount: 2500,
+    },
+    {
+      title: 'Medical Aid for Refugees',
+      description: 'Provide medical assistance to refugees in crisis zones.',
+      image:
+        'https://images.unsplash.com/photo-1581056771392-8a90ddb76831?q=80&w=2670&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D',
+      altText: 'Medical Aid',
+      targetAmount: 8000,
+      donatedAmount: 4500,
+    },
+  ],
+}
+
+const EventPage = () => {
+  const searchParams = useSearchParams()
+  const type = searchParams.get('type') // Get the donation type (e.g., 'money', 'clothes', 'food')
+
+  const campaigns = donationCampaigns[type as keyof typeof donationCampaigns] || []
+  const router = useRouter()
+
+  const handleDonateClick = (campaign: any) => {
+    const title = campaign?.title || 'default-campaign-title'
+    const formattedTitle = title.replace(/\s+/g, '-').toLowerCase()
+
+    const donationType = campaign.requiredItems ? 'items' : 'money'
+
+    const queryParams = new URLSearchParams({
+      campaign: formattedTitle,
+      donationType: donationType,
+      targetAmount: campaign.targetAmount?.toString() || '0',
+      donatedAmount: campaign.donatedAmount?.toString() || '0',
+      requiredItems: campaign.requiredItems ? JSON.stringify(campaign.requiredItems) : '',
+    })
+
+    router.push(`/donations/form?${queryParams.toString()}`)
+  }
+
+  return (
+    <Flex direction="column" align="center" gap={6} p={4}>
+      <Heading fontSize="2xl" mb={4}>
+        {type ? `Donation Campaigns for ${type}` : 'Donation Campaigns'}
```

Comment:
Missing translations, use `useTranslations()` from `next-intl` there are examples how it works in other files in this repo. This comment applies to all your files